wagtail/admin/templates/wagtailadmin/admin_base.html

```diff
@@ -43,6 +43,8 @@
             };
         })(document, window);
     </script>
+    {% wagtail_config as config %}
+    {{ config|json_script:"wagtail-config" }}
```

Comment:
I'm keen to move all the config in the `<script>` just above these lines to the new `{% wagtail_config %}` tag so that we don't directly use Django templates to put config in the JS. I've already got it working previously, but I'll make that in a separate PR.